content/04.results.md

```diff
@@ -39,27 +39,45 @@ The x-axis shows the number of randomly sampled label functions incorporated as
 The y-axis shows the area under the receiver operating curve (AUROC).
 Each point on the plot shows the average of 50 sample runs, while the error bars show the 95% confidence intervals of all runs.
 The baseline and “All” data points consist of sampling from the entire fixed set of label functions.
-](https://raw.githubusercontent.com/danich1/snorkeling-full-text/80c9c899e91418af6fba7c74c9bc28b04be78458/figure_generation/output/figure_four.png){#fig:auroc_grabbag_gen_model_test_set}
+](https://raw.githubusercontent.com/danich1/snorkeling-full-text/cd38c26db62f7eb7bc83fd9c424d0c8912512d06/figure_generation/output/figure_four.png){#fig:auroc_grabbag_gen_model_test_set}
 
 
 ### Discriminative Model Performance
 
-The discriminative model is designed to augment performance over the generative model by incorporating textual features along with estimated training labels.
-The discriminative model is a piecewise convolutional neural network trained over word embeddings (See Methods and Materials).
-We found that the discriminative model generally out-performed the generative model as more edge-specific label functions are incorporated (Figure {@fig:auroc_discriminative_model_performance} and Supplemental Figure {@fig:aupr_discriminative_model_performance}).
-The discriminative model's performance is often poorest when very few edge-specific label functions are added to the baseline model (seen in Disease associates Gene (DaG), Compound binds Gene (CbG) and Gene interacts Gene (GiG)). 
-This suggests that generative models trained with more label functions produce outputs that are more suitable for training discriminative models.
-An exception to this trend is Compound treats Disease (CtD) where the discriminative model out-performs the generative model at all levels of sampling.
-We observed the opposite trend with the Compound-binds-Gene (CbG) edges: the discriminative model was always poorer or indistinguishable from the generative model.
+The discriminative model augments performance over the generative model by incorporating textual features together with estimated training labels.
+We found that the discriminative model generally outperforms the generative model with respect to AUROC as more edge-specific label functions are incorporated (Figure {@fig:auroc_discriminative_model_performance}).
```

Comment:
When reporting actual results, it should be past tense. Probably good to revise this paragraph to fit.